### ISSUE TITLE
AttributeError: 'dict' object has no attribute 'scores'

### ISSUE THREAD

**Issue by ymoslem** [Unbabel/COMET#183]
#### What is your question?

Hello! I have an issue running COMET on Google Colab. I received the following error:

```bash
Predicting DataLoader 0: 100%|██████████| 79/79 [03:37<00:00,  2.75s/it]
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
[<ipython-input-6-82b44970d657>](https://localhost:8080/#) in <cell line: 14>()
     12 model = load_from_checkpoint(model_path)
     13 
---> 14 seg_scores, sys_score = model.predict(data, batch_size=128, gpus=1).values()
     15 comet = round(sys_score*100, 2)
     16 print("COMET:", comet)

1 frames
[/usr/local/lib/python3.10/dist-packages/comet/models/base.py](https://localhost:8080/#) in <listcomp>(.0)
    650             exit()
    651 
--> 652         scores = torch.cat([pred.scores for pred in predictions], dim=0).tolist()
    653         if "metadata" in predictions[0]:
    654             metadata = flatten_metadata([pred.metadata for pred in predictions])

AttributeError: 'dict' object has no attribute 'scores' 
```

#### Code example
```python
from comet import download_model, load_from_checkpoint

model = load_from_checkpoint(download_model("Unbabel/wmt22-comet-da"))
data = [
    {
        "src": "Ich esse gerne Pizza.",
        "mt": "I like to eat happy pizza.",
        "ref": "I like to eat pizza."
    },
]
model.predict(data, batch_size=1, gpus=0)[0]
```

#### What's your environment?

 - OS: Linux (Google Colab)
 - Python: 3.10.12
 - Packaging: pip 23.3.1
 - Version:
   - unbabel-comet 2.2.0
   - torch 2.1.1
   - torchmetrics 0.10.3
   - pytorch-lightning 2.1.2
   - transformers 4.35.2

#### What have you tried?

I have also tried it locally on macOS using a virtual environment on Python 3.9.0. I have got the same error message.

**Comment by Evstrife** [comment]
I have a same issue with you since yesterday

**Comment by ymoslem** [comment]
This can be fixed through this edit #185 

Changed:
```
scores = torch.cat([pred.scores for pred in predictions], dim=0).tolist()
```
to:
```
scores = torch.cat([pred["scores"] for pred in predictions], dim=0).tolist()
```

**Comment by mjpost** [comment]
Can we reopen this? The bug exists in released software (at least since 2.1.0), it should be fixed in a point release.
